jaxrs-spec/src/main/asciidoc/chapters/providers/_providers.adoc

```diff
@@ -20,6 +20,15 @@ and MAY be annotated with `@Provider` for automatic discovery purposes;
 the integration of pre-packaged providers into the JAX-RS runtime is
 implementation dependent.
 
+[[extensions]]
+Providers MAY implement the `Extension` interface to declare it as an
+extension of the runtime.
+
+Compatible products MUST automatically register a class in
+`jakarta.ws.rs.core.Configuration` runtime configurations if the
+class is a _provider class_ for the `Extension` _service type_ according
```

Comment:
I would say _service provider class_ to distinguish from JAX-RS providers.

Reply:
I agree

Reply:
Actually those *are* JAX-RS providers, so what problem do you see here?